graspy/pipeline/anomaly.py

```diff
@@ -0,0 +1,376 @@
+from collections import namedtuple
+
+import numpy as np
+from scipy.special import gamma
+
+from ..embed import MultipleASE, OmnibusEmbed
+from ..utils import import_multigraphs, is_almost_symmetric
+
+AnomalyResult = namedtuple(
```

Comment:
there's a better syntax for this that allows for type hinting, and since that's going to be a requirement you may want to take a stab at it going forward :)

```python
from typing import Dict, List, NamedTuple

class AnomalyResult(NamedTuple):  # I don't actually know what these types are I'm just going by name.  Which is why I like type hinting, because then I don't have to hypothesize types
    graph_anomaly_indices: List[int]
    graph_anomaly_dict: Dict[Any, Any]  
    vertex_anomaly_indices: List[int]
    vertex_anomaly_dict: Dict[Any, Any]
```

Should behave pretty much exactly the same though